### PR TITLE
Add recipe for MATAM

### PR DESCRIPTION
**Pull request by loic-couderc**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/matam/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - python >=3
+
+  run:
+    - coreutils # Centos6 (otherwise sort --parallel is unrecognized)
+    - wget
+    - bzip2
+    - zlib
+    - python >=3
+    - numpy
+    - samtools=1.9=h8571acd_11 # samtools libcrypto.so.1.0.0 error (https://github.com/sunbeam-labs/sunbeam/issues/181)
```
Comment (dpryan79) [inline recipes/matam/meta.yaml]: This should no longer be needed.

File: recipes/matam/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+    - autoconf
+    - automake
+
+  host:
+    - zlib
+    - python >=3
```
Comment (dpryan79) [inline recipes/matam/meta.yaml]: Is python actually needed for installation? I see there is one script, but that shouldn't need to be compiled.

File: recipes/matam/meta.yaml (hunk 1 above)
Comment (loic-couderc) [reply recipes/matam/meta.yaml]: You are right

File: recipes/matam/meta.yaml (hunk 2 above)
Comment (loic-couderc) [reply recipes/matam/meta.yaml]: Yes, python is not needed for the installation.

**Comment by loic-couderc** [comment]
Hi @bioconda/core, can you add @ppericard and me to bioconda ? thank you.

**Comment by dpryan79** [comment]
@loic-couderc Sure, you'll both receive invites shortly

**Pull request merged**